app/views/registrations/show.html.erb

```diff
@@ -50,6 +50,10 @@
         <% if @registration.main_people.any? %>
           <%= render "shared/registrations/main_people_details", resource: @registration %>
         <% end %>
+
+        <% if @registration.receipt_email.present? && !@registration.receipt_email.empty? %>
```

Comment:
```suggestion
        <% if @registration.receipt_email.present? %>
```

`.present?` is more thorough so should cover us!

Reply:
Awesome. Had it in my head it only covered the field being there 🤦